contracts/partials/dex_core/standalone/helpers.ligo

```diff
@@ -7,3 +7,16 @@ function get_dev_address(
   const s               : storage_t)
                         : address is
   s.dev_store.dev_address
+
+function check_is_registered_strategy(
+  const strategy        : address;
+  const s               : storage_t)
```

Comment:
Isn't it more cost effective to pass strategy_factory address only if it is not a view method?

Reply:
Do we use it somewhere?